lib/game.rb

```diff
@@ -0,0 +1,69 @@
+class Game
+
+	WIN_COMBINATIONS = [[0,4,8], [6,4,2], [0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8]]
+
+	attr_accessor :board, :player_1, :player_2
+
+		# Originally tried to make the initialize arguments keywords, but one test would not
+		# pass if this was the case.
+	 def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new )
+	 	@board = board
+			# Computer instance will need access to WIN_COMBINATIONS (via move(board)) to make decisions
+			# on how to move.  This passes WIN_COMBINATIONS to board instance.
+		board.win_combinations = self.class::WIN_COMBINATIONS
+		@player_1 = player_1
+	 	@player_2 = player_2
+	 end
+
+  def turn_count
+		board.cells.count { |square| square != " " }
+  end
+
+  def current_player
+		turn_count.even? ? player_1 : player_2
+  end
+
+	def won?
```

Comment:
Here you can use [`Enumerable#detect`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-detect) to return the winning combination or `nil` if none is found.

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end